### ISSUE TITLE
Error: name 'args_instance' is not defined" when dsconf <instance> replication monitor

### ISSUE THREAD

**Issue by definite** [389ds/389-ds-base#5850]
**Issue Description**
I ran:

~~~
dsconf -v -D 'cn=directory manager' ldap://localhost replication monitor
~~~

And input credentials like the following:

Enter password for cn=Directory Manager on ldap://localhost

Enter a bind DN for replica.example.com:389: cn=directory manager
Enter a password for cn=directory manager on replica.example.com:389: 

Then
~~~
Traceback (most recent call last):
  File "/usr/sbin/dsconf", line 138, in <module>
    result = args.func(inst, None, log, args)
  File "/usr/lib/python3.9/site-packages/lib389/cli_conf/replication.py", line 422, in get_repl_monitor_info
    report_dict = repl_monitor.generate_report(get_credentials, args.json)
  File "/usr/lib/python3.9/site-packages/lib389/replica.py", line 2708, in generate_report
    args_instance[SER_HOST] = supplier_hostname
NameError: name 'args_instance' is not defined
ERROR: Error: name 'args_instance' is not defined
~~~

**Package Version and Platform:**
 - Platform: RHEL9
 - Package and version: 389-ds-base-2.2.4-3.el9.x86_64

**Steps to Reproduce**
0. Setup environment with 389-ds replicas
1. dsconf -v -D 'cn=directory manager' ldap://localhost replication manager
2. Input credentials

Following error shown:
~~~
Traceback (most recent call last):
  File "/usr/sbin/dsconf", line 138, in <module>
    result = args.func(inst, None, log, args)
  File "/usr/lib/python3.9/site-packages/lib389/cli_conf/replication.py", line 422, in get_repl_monitor_info
    report_dict = repl_monitor.generate_report(get_credentials, args.json)
  File "/usr/lib/python3.9/site-packages/lib389/replica.py", line 2708, in generate_report
    args_instance[SER_HOST] = supplier_hostname
NameError: name 'args_instance' is not defined
ERROR: Error: name 'args_instance' is not defined
~~~

**Expected results**
No error

**Additional context**
Workaround: edit replica.py, and add args_instance to lib389._constants import like:

~~~
from lib389._constants import CONSUMER_REPLICAID, REPLICA_RDWR_TYPE, REPLICA_FLAGS_WRITE, REPLICA_RDONLY_TYPE, \
                              REPLICA_FLAGS_RDONLY, REPLICA_ID, REPLICA_TYPE, REPLICA_SUFFIX, REPLICA_BINDDN, \
                              RDN_REPLICA, REPLICA_FLAGS, REPLICA_RUV_UUID, REPLICA_OC_TOMBSTONE, DN_MAPPING_TREE, \
                              DN_CONFIG, DN_PLUGIN, REPLICATION_BIND_DN, REPLICATION_BIND_PW, ReplicaRole, \
                              args_instance, defaultProperties
~~~

RHBZ [2224930](https://bugzilla.redhat.com/show_bug.cgi?id=2224930) - "Error: name 'args_instance' is not defined" when dsconf <instance> replication monitor

**Comment by mreynolds389** [comment]
This was already fixed in https://github.com/389ds/389-ds-base/issues/5588

**Comment by definite** [comment]
Indeed, the fix is at commit 015c9abdf493688c9a5c3fa57451cca605807b31
